### PR TITLE
React refactor phase 4

### PR DESCRIPTION
**Pull request by chrisnojima**
Phase 4: Fix ios
- [x] Fixup broken go stuff
- [x] Point to shared
- [x] sendAppEventWithName -> sendDeviceEventWithName
- [x] fix bridge pointer
- [x] update wipe cache path
- [x] update ios deps
- [x] add rn-cli with shared watcher

@keybase/react-hackers this should make both sides work now

### CODE REVIEW

**Comment by chrisnojima** [comment]
Note: Uninstall watchman if you have it installed, we need react packager to fallback!

File: go/vendor/github.com/keybase/go-keychain/keychain.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// +build darwin ios
+// +build darwin
```
Comment (MarcoPolo) [inline go/vendor/github.com/keybase/go-keychain/keychain.go]: does darwin automatically include ios?

**Comment by MarcoPolo** [comment]
:+1: :bangbang: 

I merged in the kbfs actions so you might want to do the .mobile -> .native to this file: https://github.com/keybase/client/tree/master/shared/actions/kbfs

File: go/vendor/github.com/keybase/go-keychain/keychain.go (hunk 1 above)
Comment (chrisnojima) [inline go/vendor/github.com/keybase/go-keychain/keychain.go]: yeah, gobuild-ios works

**Comment by chrisnojima** [comment]
i'll merge this in the morning

**Pull request merged**